### PR TITLE
Add modules

### PR DESCRIPTION
**Pull request by ANTOINE1128**
# In this PR I have followed these project Requirements:

1 - Create a new app. Give it a meaningful name.
2 - Initialize your project with Git.
3 - Make sure that your project has Postgres database set up.
4 - Build your project schema.
5 - Create and run the necessary migration files.
6 - Table and column names should match the ERD diagram.
note: photo for the users table should be a link to an image
7 - Foreign keys should be included.
8 - Pay attention that in the posts table the column holding the foreign key to the users table should be named author_id
9 - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @ANTOINE1128,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a descriptive README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ANTOINE1128** [comment]
@TedLivist  man You were faster than me I requested a review once I clicked on request a review I was like, oh no I forgot the readme file so I tried to do it very fast before you do the review but you were faster  
anyway thank you for the review

**Review by DammyShittu — COMMENTED** [review]
Hi @ANTOINE1128,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the "Presentation" header link. It is broken.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ActiveRecord::Schema[7.0].define(version: 2023_08_23_084504) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.integer "author_id"
+    t.integer "post_id"
+    t.text "text"
+    t.datetime "createdAt"
+    t.datetime "updatedAt"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly remove these unnecessary table columns from the comments table. We already have `updated_at` and `created_at` as created from the timestamps module.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.integer "author_id"
+    t.integer "post_id"
+    t.datetime "createdAt"
+    t.datetime "updatedAt"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly remove these unnecessary table columns from the likes table. We already have `updated_at` and `created_at` as created from the timestamps module.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["post_id"], name: "index_likes_on_post_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.integer "author_id"
+    t.string "title"
+    t.text "text"
+    t.datetime "createdAt"
+    t.datetime "updatedAt"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly remove these unnecessary table columns from the posts table. We already have `updated_at` and `created_at` as created from the timestamps module.

File: db/schema.rb [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_posts_on_author_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.text "bio"
+    t.datetime "createdAt"
+    t.datetime "updatedAt"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly remove these unnecessary table columns from the users table. We already have `updated_at` and `created_at` as created from the timestamps module.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ANTOINE1128,

Good job so far!
✔️ Correct column types for database columns
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @ANTOINE1128    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Database  setup correctly
- [x] Professional Readme




 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**